src/main/groovy/io/cify/framework/core/Device.groovy

```diff
@@ -203,6 +213,12 @@ class Device implements IDevice {
                 stopRecording()
```

Comment:
one line above: what is the purpose to take a screenshot sinnce if video recording was switched on the last screen state was captured by video.

Reply:
I'm not sure. Might be a business requirement